### ISSUE TITLE
OCS Impact Research UXR Onboarding

### ISSUE THREAD

**Issue by mkleibman** [hackforla/open-community-survey#103]
### Overview
Create a UXR onboarding guide for OCS Impact Research. Please refer to the [TDM Calculator](https://docs.google.com/presentation/d/1VUTxh5eR0PHUAhaU8ZgNlsnrj_qmaBXcv9IUmfR0aTs/edit?usp=sharing) guide for reference.

### Action Items

- [ ] Create a UXR Onboarding Guide Template using Google Slides.
- [ ] Create a background image for the OCS Guide + Template to be used in interviews (see TDM Calculator project for reference). Also make sure to have this in the UXR onboarding slideshow.
- [ ] Review with research team to determine language, speaker notes
- [ ] Review design assets with research team to determine appropriate tone etc.
- [ ] Conduct mock interviews 

### Resources/Instructions
At Issue:
[OCS Impact Research UXR Onboarding [TEMPLATE]](https://docs.google.com/presentation/d/1ko1pmgrw0kxGU1fhwxHVUIomWMhWL_CgddRBSrtAbQo/edit?usp=sharing)
For reference:
[TDM Calculator UXR Onboarding - Second Round [TEMPLATE]](https://docs.google.com/presentation/d/1VUTxh5eR0PHUAhaU8ZgNlsnrj_qmaBXcv9IUmfR0aTs/edit?usp=sharing)
OCS Project Templates:
[OCS Slide Template](https://docs.google.com/presentation/d/1OsP8zHxaRdAwAaHDL5j7hy8ts9jEowqobKg9Ou1a4qw/edit?usp=sharing)
[OCS Communication Guide](https://docs.google.com/presentation/d/1V_JVpqLadtYUQDdDfVUmYmgt-nCfvz_tNTFkWmbNeG0/edit?usp=sharing)

**Comment by mkleibman** [comment]
@kalyaniraman please review at your leisure and let me know of any changes. Thanks!

**Comment by kalyaniraman** [comment]
@mkleibman Issue looks good. Do you want to move the reference link to TDM along with other reference links at the bottom of the issue? Also for Milestones, i selected Guides option. Let me know if you think it should belong with another milestone.

**Comment by mkleibman** [comment]
Yes TDM can go along with other links. I will continue on this today.

**Comment by mkleibman** [comment]
Please see updated [slide designs](https://docs.google.com/presentation/d/1ko1pmgrw0kxGU1fhwxHVUIomWMhWL_CgddRBSrtAbQo/edit#slide=id.g117ba7639ae_0_16). Ignore images that are hidden, as they have not been checked (pending further Qs about how to proceed).

I have revised typefaces, cleaned up alignments, added vector graphics and inserted placeholder text. I also added comments for next steps to discuss next meeting.

Tagging: @LAStorm & @kalyaniraman for feedback when available.

**Comment by mkleibman** [comment]
Confirming that I will begin work on this today, and complete through the weekend ahead on Monday's meeting.
This also pertains to #96 

I will update this comment with my notes shortly.

**Comment by mkleibman** [comment]
Please use stars to vote on preferred background image in Figma. Leave a comment for any improvements you would like to see, or additional options not offered. 

[OCS Figma Page: Open Source Images](https://www.figma.com/file/04T31AqsprIIkuBN4OfwOo/Open-Community-Survey-Designs?node-id=108%3A294)

**Comment by mkleibman** [comment]
Per Meeting 5/9

- [ ] Add EasyRetro slide info
- [ ] Finalize mock interview slides 
- [ ] Finalize BG image (see #107  and place in Drive)

Outstanding Qs:

1. Do we have an interview script finalized? If so, can we add it to this issue for reference?
2. What is the status on a C.U.T guide for this project?

**Comment by kalyaniraman** [comment]
Please refer to the Internship project guide and update ours to be similar to that teams deck

- [Internship Research Training Deck](https://github.com/hackforla/internship/wiki/Research-Plan-6:-Intern-Intake-Interviews)

**Comment by ambersu123** [comment]
@ExperimentsInHonesty do we want to copy the same styling as well just with our branded colors and fonts

**Comment by mkleibman** [comment]
Please review slides to confirm they match expectations using Intern guide and let me know of any changes I should make. I am posting here since our meeting did not go forward today and next week is PMs only (although I am happy to participate if you think time is of the essence).
[Our slide deck](https://docs.google.com/presentation/d/1ko1pmgrw0kxGU1fhwxHVUIomWMhWL_CgddRBSrtAbQo/edit#slide=id.g1283eb88107_0_0)

[Intern deck for which it is based](https://docs.google.com/presentation/d/1kAo2QSam5rCAsffk14PQb-lqkVZykCNqsU_kHZQXmxY/edit#slide=id.g133ea43999e_0_8)

cc/ @kalyaniraman @ambersu123 @ExperimentsInHonesty

**Comment by ExperimentsInHonesty** [comment]
Hi @mkleibman 

1. I made copy of the session table and role call document that is referenced in the training, calling it [OCS: Interview Session and Roll Call Table template](https://docs.google.com/spreadsheets/d/1htY3-tWNPUYqseF39vgoRKjmFzvn6Ehv21nMfsS8Q_s/edit#gid=1585831764) and putting it in our communications guide folder.  It will need to be customized to our project (change out the logo) and then a copy made for this specific deck.

2. Can you change out the red bullets for one of our colors
<details>
<summary>see this animated gif</summary>

![bullet_recording](https://user-images.githubusercontent.com/37763229/178115550-689a8f27-0a1b-4779-b993-3488dd6ecb58.gif)
<details>

**Comment by mkleibman** [comment]
> @mkleibman Please provide update
> 
> 1. Progress: "What is the current status of your project? What have you completed and what is left to do?"
> 2. Blockers: "Difficulties or errors encountered."
> 3. Availability: "How much time will you have this week to work on this issue?"
> 4. ETA: "When do you expect this issue to be completed?"
> 5. Pictures (if necessary): "Add any pictures that will help illustrate what you are working on."

1. Changed colors of bullet points
2. I don't have team specific links to update
3. I am now widely available and back from vacation
4. It's a 1pt size once I have links (I can create or setup if needed)
5. [See link ](https://docs.google.com/presentation/d/1ko1pmgrw0kxGU1fhwxHVUIomWMhWL_CgddRBSrtAbQo/edit#slide=id.g137a1689c41_2_86)

**Comment by ExperimentsInHonesty** [comment]
- an example of an interview guide (sometimes known as cut guide) [TWE RP6 Intern Intake Interview Instructions](https://docs.google.com/document/d/1UwjsKQcVc3ZQRCaePmygM4o-b9paSEsbczTZV0RFucY)

**Comment by LAStorm** [comment]
[Here](https://docs.google.com/document/d/1ZAsX84xbaj2mCn38NXJIweAClwOvuItsvzt26-maGdk/edit?usp=sharing) is the link to the interview guide.

**Comment by mkleibman** [comment]
Notes 8/22

- [ ] Create links for training deck
- [ ] Confirm what style vectors to use; [the people-based graphics discussed in earlier meetings - at bottom of doc-](https://docs.google.com/document/d/1M09IwSGaESn9OB_CBh5qn1QLeG1YZ1ATRxvVrjHYp6U/edit) OR the simple shapes used in the intern deck but in our colors

**Comment by mkleibman** [comment]
Do we have an OCS account email for EasyRetro so I can create new boards?
These remain the only links I have not set up for OCS, converted from TWE. I need them both for the slide and instructions doc. Alternately, if we have a password doc or access to an email, I can use that to set up a new EasyRetro for the project.

cc/ @kalyaniraman @ExperimentsInHonesty

**Comment by mkleibman** [comment]
Good morning,
I cannot sign into our OCS GMail account since it notes the password was changed 5 months ago.
Can someone please update the password in 1Password in the Open Community Survey vault or provide it to me so I can?
I cannot create an EasyRetro board until I verify the email; so alternately if anyone has access, please verify.

cc/ @ExperimentsInHonesty @LAStorm -- do either of you have current PW?

**Comment by mkleibman** [comment]
Updates:

- [ ] Create Easy Retro boards for OCS and link them in slides and training instructions
- [ ] Split Easy Retro and Instructions into two separate slides with Easy Retro first
- [ ] Reduce bullet points or entirely remove existing instructions slide IF redundant to instructions doc
- [ ] Replace all instances of intern to participants 
- [ ] Adjust/fix roll call header logo

**Comment by mkleibman** [comment]
- [ ] Complete Figma letter and report form for Google docs

**Comment by joshfishman** [comment]
Reviewed Slide Deck Template 

Slide 5 - Project Description
This slide needs to be rewritten, the current description is describing two of our many projects

> 
The Open Community Survey project creates transparent reports supported by a direct collection of personal perspectives from LA residents to help The LA Department of Neighborhood Empowerment (empowerla.org) and the Los Angeles Neighborhood Councils (NCs) to understand how constituents are interacting with, and what they need from, their websites.
